### ISSUE TITLE
Runtime: Debug `Input is invalid/unsupported GZIP stream`

### ISSUE THREAD

**Issue by begelundmuller** [rilldata/rill#2728]
See initial discovery here: https://rilldata.slack.com/archives/C01190F1R1D/p1688375697799989?thread_ts=1688318625.365559&cid=C01190F1R1D

**Comment by begelundmuller** [comment]
Sharing a snippet from seeing this and a (probably related) DuckDB internal error during ingestion of a BigQuery source:

```
{"level":"info","ts":1691668154.846046,"msg":"Ingesting source \"gke_metering\""}
{"level":"info","ts":1691668170.9113238,"msg":"records to be ingested","rows":128282364}
{"level":"error","ts":1691668173.589304,"msg":"Ingestion failed for \"gke_metering\" : Input is invalid/unsupported GZIP stream"}
{"level":"info","ts":1691668173.589987,"msg":"reconcile completed","result":{"AddedObjects":[],"UpdatedObjects":[{"Name":"gke_metering","Type":2,"Object":{"name":"gke_metering","connector":"bigquery","properties":{"project_id":"rilldata","sql":"select cluster_location, cluster_name, namespace, resource_name, sku_id, start_time, end_time from `gke_metering.gke_cluster_resource_consumption`"}},"Path":"/sources/gke_metering.yaml","Embedded":false,"BytesIngested":0,"Parents":null,"Children":null,"CreatedOn":"0001-01-01T00:00:00Z","UpdatedOn":"2023-08-10T13:49:14.839034271+02:00","RefreshedOn":"0001-01-01T00:00:00Z"}],"DroppedObjects":[],"AffectedPaths":["/sources/gke_metering.yaml"],"Errors":[{"code":4,"message":"Input is invalid/unsupported GZIP stream","file_path":"/sources/gke_metering.yaml"}]}}
{"level":"error","ts":1691668173.601183,"msg":"encountered internal DuckDB error - scheduling reopen of DuckDB","error":"INTERNAL Error: Could not find node in column segment tree!\nAttempting to find row number \"0\" in 0 nodes\n"}
{"level":"info","ts":1691668173.6037009,"msg":"reopened DuckDB successfully"}
{"level":"error","ts":1691668173.675425,"msg":"encountered internal DuckDB error - scheduling reopen of DuckDB","error":"INTERNAL Error: Could not find node in column segment tree!\nAttempting to find row number \"0\" in 0 nodes\n"}
{"level":"info","ts":1691668173.67804,"msg":"reopened DuckDB successfully"}
{"level":"info","ts":1691668191.300719,"msg":"Rill shutdown gracefully"}
```

**Comment by begelundmuller** [comment]
The DuckDB issue has been closed and the issue should be resolved when we upgrade to DuckDB 0.9
